components/Handlers.js

```diff
@@ -0,0 +1,45 @@
+import { render } from '@asyncapi/generator-react-sdk';
+import { pascalCase } from './common';
+
+
+let subscriptionFunction = (channelName, operation, message) => `
+//${operation} subscription handler for ${channelName}        
+func ${operation}(msg *message.Message) error {
+    log.Printf("received message payload: %s", string(msg.Payload))
+
+    var lm payloads.${message}
+    err := json.Unmarshal(msg.Payload, &lm)
+    if err != nil {
+        log.Fatalf("error unmarshalling message: %s, err is: %s", msg.Payload, err)
+        return err
```

Comment:
This is not gonna be executed ever if you run a `log.fatal` before. `log.Fatal` terminates the execution of the program.